bcrs-ui/.env.production

```diff
@@ -0,0 +1,2 @@
+VUE_APP_PATH = basePath
```

Comment:
Perfect content for now.
Please also check in `.env`.

Reply:
Oh right, i made it. It must have been ignored!